guides/source/action_text_overview.md

```diff
@@ -8,149 +8,141 @@ rich text content.
 
 After reading this guide, you will know:
 
-* How to configure Action Text.
-* How to handle rich text content.
-* How to style rich text content and attachments.
+* What action text is and how to install and configure Action Text.
+* How to create, render, style and customize rich text content.
+* How to handle attachments.
 
 --------------------------------------------------------------------------------
 
 What is Action Text?
 --------------------
 
-Action Text brings rich text content and editing to Rails. It includes
-the [Trix editor](https://trix-editor.org) that handles everything from formatting
-to links to quotes to lists to embedded images and galleries.
-The rich text content generated by the Trix editor is saved in its own
-RichText model that can be associated with any existing Active Record model in the application.
-Any embedded images (or other attachments) are automatically stored using
-Active Storage and associated with the included RichText model.
+Action Text facilitates the handling and display of Rich Text Content. Rich Text content is text that includes formatting elements such as bold, italics, colors, and hyperlinks, providing a visually enhanced and structured presentation beyond plain text. It allows us to create rich text content, store it in a table, then attach it to any of our models.
 
-When it's time to render the content, Action Text processes it by sanitizing its
-content so that it's safe to embed directly into the page's HTML.
+**The Trix Editor**
```

Comment:
Maybe this can be a NOTE instead of body text, it could work as well. Any thoughts?

Reply:
I like it. Let's try it.